test/functional/rpc_createmultisig.py

```diff
@@ -17,29 +21,39 @@ def skip_test_if_missing_module(self):
 
     def get_keys(self):
         node0, node1, node2 = self.nodes
-        self.add = [node1.getnewaddress() for _ in range(self.nkeys)]
-        self.pub = [node1.getaddressinfo(a)["pubkey"] for a in self.add]
-        self.priv = [node1.dumpprivkey(a) for a in self.add]
+        add = [node1.getnewaddress() for _ in range(self.nkeys)]
+        self.pub = [node1.getaddressinfo(a)["pubkey"] for a in add]
+        self.priv = [node1.dumpprivkey(a) for a in add]
         self.final = node2.getnewaddress()
 
     def run_test(self):
-        node0,node1,node2 = self.nodes
+        node0, node1, node2 = self.nodes
+
+        self.check_createmultisig_errors()
```

Comment:
Perhaps an info log for the new test "Test addmultisigaddress fails for watchonly addresses" and one to replace the comment "# 50 BTC each, rest will be 25 BTC each" with something clearer.

Current output:
```
$ test/functional/rpc_createmultisig.py 
2019-04-17T10:26:29.745000Z TestFramework (INFO): Initializing test directory /tmp/bitcoin_func_test_chsfh_ip
2019-04-17T10:26:32.375000Z TestFramework (INFO): n/m=2/3 bech32 size=337 vsize=147 weight=586
2019-04-17T10:26:32.581000Z TestFramework (INFO): n/m=2/3 p2sh-segwit size=372 vsize=182 weight=726
2019-04-17T10:26:32.797000Z TestFramework (INFO): n/m=2/3 legacy size=335 vsize=335 weight=1340
2019-04-17T10:26:33.053000Z TestFramework (INFO): n/m=3/3 bech32 size=409 vsize=165 weight=658
2019-04-17T10:26:33.339000Z TestFramework (INFO): n/m=3/3 p2sh-segwit size=444 vsize=200 weight=798
2019-04-17T10:26:33.606000Z TestFramework (INFO): n/m=3/3 legacy size=409 vsize=409 weight=1636
2019-04-17T10:26:33.931000Z TestFramework (INFO): n/m=2/5 bech32 size=405 vsize=164 weight=654
2019-04-17T10:26:34.253000Z TestFramework (INFO): n/m=2/5 p2sh-segwit size=440 vsize=199 weight=794
2019-04-17T10:26:34.581000Z TestFramework (INFO): n/m=2/5 legacy size=405 vsize=405 weight=1620
2019-04-17T10:26:34.872000Z TestFramework (INFO): n/m=3/5 bech32 size=477 vsize=182 weight=726
2019-04-17T10:26:35.187000Z TestFramework (INFO): n/m=3/5 p2sh-segwit size=512 vsize=217 weight=866
2019-04-17T10:26:35.517000Z TestFramework (INFO): n/m=3/5 legacy size=477 vsize=477 weight=1908
2019-04-17T10:26:37.090000Z TestFramework (INFO): Stopping nodes
2019-04-17T10:26:37.495000Z TestFramework (INFO): Cleaning up /tmp/bitcoin_func_test_chsfh_ip on exit
2019-04-17T10:26:37.495000Z TestFramework (INFO): Tests successful
```